test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -1809,4 +1809,93 @@ describe('v1.0.0 to v2.0.0', function () {
             });
         });
     });
+
+    describe('retainEmptyValues', function () {
+        var options = {
+            inputVersion: '1.0.0',
+            outputVersion: '2.0.0',
+            retainIds: true,
+            retainEmptyValues: true
+        };
+
+        it('should nullify empty descriptions when set to true', function () {
+            transformer.convert({
+                id: '9ac7325c-cc3f-4c20-b0f8-a435766cb74c',
+                description: '', // this represents the case where descriptions are removed
+                folders: [{
+                    id: 'f3285fa0-e361-43ba-ba15-618c7a911e84',
+                    description: null,
+                    order: ['9d123ce5-314a-40cd-9852-6a8569513f4e']
+                }],
+                requests: [{
+                    id: '9d123ce5-314a-40cd-9852-6a8569513f4e',
```

Comment:
path variables need to be added to this test case

Reply:
Added